sass/px-dropdown.scss

```diff
@@ -98,14 +90,22 @@ $inuit-enable-btn--bare--primary: true;
   }
   &[disabled] {
     color: var(--px-dropdown-text-color--disabled, $gray4);
+    pointer-events: none;
     cursor: not-allowed;
   }
 }
+.dropdown-option__item{
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  pointer-events: none;
```

Comment:
It was the only way I could get this to lay out right. Wade didn't want the items in the dropdown to grow but rather ellipsis. 
![screen shot 2017-06-05 at 11 28 15 am](https://cloud.githubusercontent.com/assets/14931112/26797571/2982efe0-49e2-11e7-904a-9b0a50c95b1c.png)